app/controllers/contact_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
         GroupMailer.anonymous_mail(mail_array, contact_params[:body], contact_params[:title]).deliver_now
       end
 
-     redirect_to :contact_index
+      redirect_to :contact_index, flash: {notice: "your mail has been sent to " + contact_params[:to_whom].first}
```

Comment:
The array `contact_params[:to_whom]` contains just the email address of the reciever (i.e. Styrit@chalmers.it when emailing styrit) and two empty strings. You can try to print it out yourself with `p contact_params[:to_whom].to_s` to check you self :)